mopidy/files/ext.conf

```diff
@@ -0,0 +1,8 @@
+[files]
+enabled = true
+media_dir =
+	  ~/:Home
+	  /data/music/music_data:Music
```

Comment:
`$XDG_MUSIC_DIR` is available if you make sure to use `mopidy.internal.path.expand_path` on the paths. Would make more sense to use this than to have a non-standard location as default.